diff --color -ur joystick_drivers-release-release-noetic-joy-1.15.1-1/CMakeLists.txt joystick_drivers-release-release-noetic-joy-1.15.1-1_new/CMakeLists.txt
--- joystick_drivers-release-release-noetic-joy-1.15.1-1/CMakeLists.txt	2021-06-07 23:00:52.000000000 +0200
+++ joystick_drivers-release-release-noetic-joy-1.15.1-1_new/CMakeLists.txt	2023-01-09 19:31:05.592105817 +0100
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.5)
 project(joy)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 # Load catkin and all dependencies required for this package
