diff -ur geometry2-release-release-humble-tf2_eigen_kdl-0.25.2-1/CMakeLists.txt geometry2-release-release-humble-tf2_eigen_kdl-0.25.2-1-new/CMakeLists.txt
--- geometry2-release-release-humble-tf2_eigen_kdl-0.25.2-1/CMakeLists.txt	2023-01-10 14:53:48.000000000 +0100
+++ geometry2-release-release-humble-tf2_eigen_kdl-0.25.2-1-new/CMakeLists.txt	2023-03-05 19:25:46.679638215 +0100
@@ -15,7 +15,6 @@
 if(NOT Eigen3_FOUND)
   find_package(Eigen3 REQUIRED)
 endif()
-find_package(orocos_kdl_vendor REQUIRED)
 find_package(orocos_kdl REQUIRED)
 find_package(tf2 REQUIRED)
 
@@ -49,7 +48,6 @@
 
 ament_export_dependencies(
   Eigen3
-  orocos_kdl_vendor
   orocos_kdl
   tf2
 )
