diff -ur geometry2-release-release-humble-tf2_geometry_msgs-0.25.2-1/CMakeLists.txt geometry2-release-release-humble-tf2_geometry_msgs-0.25.2-1-new/CMakeLists.txt
--- geometry2-release-release-humble-tf2_geometry_msgs-0.25.2-1/CMakeLists.txt	2023-01-10 14:53:44.000000000 +0100
+++ geometry2-release-release-humble-tf2_geometry_msgs-0.25.2-1-new/CMakeLists.txt	2023-03-05 20:19:24.756237246 +0100
@@ -19,7 +19,6 @@
 find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
 
 find_package(geometry_msgs REQUIRED)
-find_package(orocos_kdl_vendor REQUIRED)
 find_package(orocos_kdl REQUIRED)
 find_package(tf2 REQUIRED)
 find_package(tf2_ros REQUIRED)
@@ -70,7 +69,6 @@
 ament_export_targets(export_${PROJECT_NAME})
 ament_export_dependencies(
   "geometry_msgs"
-  "orocos_kdl_vendor"
   "orocos_kdl"
   "tf2"
   "tf2_ros")
diff -ur geometry2-release-release-humble-tf2_geometry_msgs-0.25.2-1/package.xml geometry2-release-release-humble-tf2_geometry_msgs-0.25.2-1-new/package.xml
--- geometry2-release-release-humble-tf2_geometry_msgs-0.25.2-1/package.xml	2023-01-10 14:53:44.000000000 +0100
+++ geometry2-release-release-humble-tf2_geometry_msgs-0.25.2-1-new/package.xml	2023-03-05 20:19:44.805315383 +0100
@@ -15,7 +15,7 @@
   <buildtool_depend>python_cmake_module</buildtool_depend>
 
   <depend>geometry_msgs</depend>
-  <depend>orocos_kdl_vendor</depend>
+  <depend>orocos_kdl</depend>
   <depend>tf2</depend>
   <depend>tf2_ros</depend>
 
