diff -ur webots_ros2-release-release-humble-webots_ros2_driver-2023.1.0-1/CMakeLists.txt webots_ros2-release-release-humble-webots_ros2_driver-2023.1.0-1_new/CMakeLists.txt
--- webots_ros2-release-release-humble-webots_ros2_driver-2023.1.0-1/CMakeLists.txt	2023-06-29 11:14:04.000000000 +0200
+++ webots_ros2-release-release-humble-webots_ros2_driver-2023.1.0-1_new/CMakeLists.txt	2023-07-03 08:22:31.693881109 +0200
@@ -28,7 +28,7 @@
 find_package(webots_ros2_msgs REQUIRED)
 find_package(tinyxml2_vendor REQUIRED)
 find_package(TinyXML2 REQUIRED)
-find_package(PythonLibs 3.10 EXACT REQUIRED)
+find_package(PythonLibs REQUIRED)
 
 add_custom_target(compile-lib-controller ALL
   COMMAND ${CMAKE_COMMAND} -E env "WEBOTS_HOME=${CMAKE_CURRENT_SOURCE_DIR}/webots" make release -f Makefile > /dev/null 2>&1
