diff --git navigation2-release-release-humble-nav2_map_server-1.1.6-1/CMakeLists.txt navigation2-release-release-humble-nav2_map_server-1.1.6-1/CMakeLists.txt
index c24e5e5c..88f01e9e 100644
--- navigation2-release-release-humble-nav2_map_server-1.1.6-1/CMakeLists.txt
+++ navigation2-release-release-humble-nav2_map_server-1.1.6-1/CMakeLists.txt
@@ -10,7 +10,7 @@ find_package(rclcpp_lifecycle REQUIRED)
 find_package(rclcpp_components REQUIRED)
 find_package(nav_msgs REQUIRED)
 find_package(nav2_msgs REQUIRED)
-find_package(yaml_cpp_vendor REQUIRED)
+find_package(yaml-cpp REQUIRED)
 find_package(std_msgs REQUIRED)
 find_package(tf2 REQUIRED)
 find_package(nav2_util REQUIRED)
@@ -54,7 +54,7 @@ add_library(${library_name} SHARED
   src/costmap_filter_info/costmap_filter_info_server.cpp)
 
 set(map_io_dependencies
-  yaml_cpp_vendor
+  yaml-cpp
   nav_msgs
   nav2_util
   tf2)
@@ -65,7 +65,7 @@ set(map_server_dependencies
   rclcpp_components
   nav_msgs
   nav2_msgs
-  yaml_cpp_vendor
+  yaml-cpp
   std_msgs
   nav2_util)
 
@@ -90,7 +90,7 @@ ament_target_dependencies(${costmap_filter_info_server_executable}
 
 ament_target_dependencies(${library_name}
   ${map_server_dependencies})
-
+target_link_libraries(${map_io_library_name} yaml-cpp)
 ament_target_dependencies(${map_io_library_name}
   ${map_io_dependencies})
 
diff --git navigation2-release-release-humble-nav2_map_server-1.1.6-1/package.xml navigation2-release-release-humble-nav2_map_server-1.1.6-1/package.xml
index 523e73c4..cfcbe6dc 100644
--- navigation2-release-release-humble-nav2_map_server-1.1.6-1/package.xml
+++ navigation2-release-release-humble-nav2_map_server-1.1.6-1/package.xml
@@ -18,8 +18,8 @@
   <depend>nav_msgs</depend>
   <depend>std_msgs</depend>
   <depend>rclcpp</depend>
-  <depend>yaml_cpp_vendor</depend>
   <depend>launch_ros</depend>
+  <depend>yaml-cpp</depend>
   <depend>launch_testing</depend>
   <depend>tf2</depend>
   <depend>nav2_msgs</depend>

