diff -ur realsense-ros-release-release-humble-realsense2_camera-4.51.1-1/CMakeLists.txt realsense-ros-release-release-humble-realsense2_camera-4.51.1-1-new/CMakeLists.txt
--- realsense-ros-release-release-humble-realsense2_camera-4.51.1-1/CMakeLists.txt	2022-09-21 09:14:52.000000000 +0200
+++ realsense-ros-release-release-humble-realsense2_camera-4.51.1-1-new/CMakeLists.txt	2023-05-04 12:12:39.370207535 +0200
@@ -101,7 +101,7 @@
 find_package(tf2 REQUIRED)
 find_package(diagnostic_updater REQUIRED)
 
-find_package(realsense2 2.51.1)
+find_package(realsense2)
 if(NOT realsense2_FOUND)
     message(FATAL_ERROR "\n\n Intel RealSense SDK 2.0 is missing, please install it from https://github.com/IntelRealSense/librealsense/releases\n\n")
 endif()
