diff -ur rviz-release-release-humble-rviz_common-11.2.5-1/CMakeLists.txt rviz-release-release-humble-rviz_common-11.2.5-1-new/CMakeLists.txt
--- rviz-release-release-humble-rviz_common-11.2.5-1/CMakeLists.txt	2023-01-10 15:06:13.000000000 +0100
+++ rviz-release-release-humble-rviz_common-11.2.5-1-new/CMakeLists.txt	2023-03-05 23:17:57.611504917 +0100
@@ -45,7 +45,7 @@
 find_package(tf2_ros REQUIRED)
 find_package(message_filters REQUIRED)
 find_package(urdf REQUIRED)
-find_package(yaml_cpp_vendor REQUIRED)
+find_package(yaml-cpp REQUIRED)
 
 find_package(TinyXML2 REQUIRED)  # provided by tinyxml_vendor
 
@@ -236,6 +236,7 @@
   rviz_ogre_vendor::OgreMain
   rviz_ogre_vendor::OgreOverlay
   Qt5::Widgets
+  yaml-cpp
 )
 
 ament_target_dependencies(rviz_common
@@ -253,7 +254,7 @@
   message_filters
   tinyxml2_vendor
   urdf
-  yaml_cpp_vendor
+  yaml-cpp
 )
 # Causes the visibility macros to use dllexport rather than dllimport,
 # which is appropriate when building the dll but not consuming it.
@@ -270,7 +271,7 @@
   tf2_geometry_msgs
   tf2_ros
   urdf
-  yaml_cpp_vendor
+  yaml-cpp
 )
 
 # Export old-style CMake variables
@@ -441,7 +442,7 @@
     ${SKIP_DISPLAY_TESTS})
   if(TARGET rviz_common_display_test)
     target_link_libraries(rviz_common_display_test rviz_common Qt5::Widgets)
-    ament_target_dependencies(rviz_common_display_test yaml_cpp_vendor)
+    ament_target_dependencies(rviz_common_display_test yaml-cpp)
   endif()
 endif()
 
diff -ur rviz-release-release-humble-rviz_common-11.2.5-1/package.xml rviz-release-release-humble-rviz_common-11.2.5-1-new/package.xml
--- rviz-release-release-humble-rviz_common-11.2.5-1/package.xml	2023-01-10 15:06:13.000000000 +0100
+++ rviz-release-release-humble-rviz_common-11.2.5-1-new/package.xml	2023-03-05 21:03:42.169882317 +0100
@@ -46,7 +46,7 @@
   <depend>message_filters</depend>
   <depend>tinyxml2_vendor</depend>
   <depend>urdf</depend>
-  <depend>yaml_cpp_vendor</depend>
+  <depend>yaml-cpp</depend>
 
   <!-- TODO(jacobperron): Replace with ament_lint_common when ament_copyright is working -->
   <test_depend>ament_cmake_cppcheck</test_depend>
