diff -u moveit_resources-release-release-noetic-moveit_resources_prbt_ikfast_manipulator_plugin-0.8.2-1/CMakeLists.txt moveit_resources-release-release-noetic-moveit_resources_prbt_ikfast_manipulator_plugin-0.8.2-1-new/CMakeLists.txt
--- moveit_resources-release-release-noetic-moveit_resources_prbt_ikfast_manipulator_plugin-0.8.2-1/CMakeLists.txt	2021-11-18 21:13:07.000000000 +0100
+++ moveit_resources-release-release-noetic-moveit_resources_prbt_ikfast_manipulator_plugin-0.8.2-1-new/CMakeLists.txt	2023-03-03 20:22:17.344946269 +0100
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.1.3)
 project(moveit_resources_prbt_ikfast_manipulator_plugin)
 
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
 add_compile_options(-Wall)
Common subdirectories: moveit_resources-release-release-noetic-moveit_resources_prbt_ikfast_manipulator_plugin-0.8.2-1/include and moveit_resources-release-release-noetic-moveit_resources_prbt_ikfast_manipulator_plugin-0.8.2-1-new/include
Common subdirectories: moveit_resources-release-release-noetic-moveit_resources_prbt_ikfast_manipulator_plugin-0.8.2-1/src and moveit_resources-release-release-noetic-moveit_resources_prbt_ikfast_manipulator_plugin-0.8.2-1-new/src
