diff -ur kdl_parser-release-release-humble-kdl_parser-2.6.4-1/CMakeLists.txt kdl_parser-release-release-humble-kdl_parser-2.6.4-1-new/CMakeLists.txt
--- kdl_parser-release-release-humble-kdl_parser-2.6.4-1/CMakeLists.txt	2023-01-10 15:16:22.000000000 +0100
+++ kdl_parser-release-release-humble-kdl_parser-2.6.4-1-new/CMakeLists.txt	2023-03-05 15:36:47.090500784 +0100
@@ -3,7 +3,6 @@
 project(kdl_parser)
 
 find_package(ament_cmake_ros REQUIRED)
-find_package(orocos_kdl_vendor REQUIRED)
 find_package(orocos_kdl REQUIRED)
 find_package(urdf REQUIRED)
 find_package(urdfdom_headers REQUIRED)
@@ -48,7 +49,6 @@
   ament_lint_auto_find_test_dependencies()
 endif()
 
-ament_export_dependencies(orocos_kdl_vendor)
 ament_export_dependencies(orocos_kdl)
 ament_export_dependencies(rcutils)
 ament_export_dependencies(urdfdom_headers)
