diff --color -ur rosidl-release-release-jazzy-rosidl_adapter-4.6.4-1/rosidl_adapter/resource/__init__.py rosidl-release-release-jazzy-rosidl_adapter-4.6.4-1-new/rosidl_adapter/resource/__init__.py
--- rosidl-release-release-jazzy-rosidl_adapter-4.6.4-1/rosidl_adapter/resource/__init__.py	2024-09-06 19:06:01.000000000 +0200
+++ rosidl-release-release-jazzy-rosidl_adapter-4.6.4-1-new/rosidl_adapter/resource/__init__.py	2024-11-10 11:30:44.244549581 +0100
@@ -18,6 +18,11 @@
 
 import em
 
+try:
+    from em import Configuration
+    em_has_configuration = True
+except ImportError:
+    em_has_configuration = False
 
 def expand_template(template_name, data, output_file, encoding='utf-8'):
     content = evaluate_template(template_name, data)
@@ -45,18 +50,33 @@
 
     output = StringIO()
     try:
-        _interpreter = em.Interpreter(
-            output=output,
-            options={
-                em.BUFFERED_OPT: True,
-                em.RAW_OPT: True,
-            })
+        if em_has_configuration:
+            config = Configuration(
+                defaultRoot=template_path,
+                defaultStdout=output,
+                deleteOnError=True,
+                rawErrors=True,
+                useProxy=True)
+            _interpreter = em.Interpreter(
+                config=config,
+                dispatcher=False)
+        else:
+            _interpreter = em.Interpreter(
+                output=output,
+                options={
+                    em.BUFFERED_OPT: True,
+                    em.RAW_OPT: True,
+                })
 
         with open(template_path, 'r') as h:
             content = h.read()
         _interpreter.invoke(
             'beforeFile', name=template_name, file=h, locals=data)
-        _interpreter.string(content, template_path, locals=data)
+        if em_has_configuration:
+            _interpreter.string(content, locals=data)
+        else:
+            _interpreter.string(content, template_path, locals=data)
+
         _interpreter.invoke('afterFile')
 
         return output.getvalue()
@@ -66,7 +86,8 @@
             file=sys.stderr)
         raise
     finally:
-        _interpreter.shutdown()
+        if _interpreter is not None:
+            _interpreter.shutdown()
         _interpreter = None
 
 
@@ -78,7 +99,10 @@
             'beforeInclude', name=template_path, file=h, locals=kwargs)
         content = h.read()
     try:
-        _interpreter.string(content, template_path, kwargs)
+        if em_has_configuration:
+            _interpreter.string(content, locals=kwargs)
+        else:
+            _interpreter.string(content, template_path, kwargs)
     except Exception as e:  # noqa: F841
         print(
             f"{e.__class__.__name__} processing template '{template_name}': "
