diff --color -ur fastrtps-release-release-jazzy-fastrtps-2.14.4-1/src/cpp/fastdds/topic/DDSSQLFilter/DDSFilterCompoundCondition.hpp fastrtps-release-release-jazzy-fastrtps-2.14.4-1-new/src/cpp/fastdds/topic/DDSSQLFilter/DDSFilterCompoundCondition.hpp
--- fastrtps-release-release-jazzy-fastrtps-2.14.4-1/src/cpp/fastdds/topic/DDSSQLFilter/DDSFilterCompoundCondition.hpp	2025-04-11 23:08:51.190090051 +0200
+++ fastrtps-release-release-jazzy-fastrtps-2.14.4-1-new/src/cpp/fastdds/topic/DDSSQLFilter/DDSFilterCompoundCondition.hpp	2025-04-11 23:09:29.046547556 +0200
@@ -21,6 +21,8 @@
 
 #include <memory>
 
+#include <cstdint>
+
 #include "DDSFilterCondition.hpp"
 
 namespace eprosima {
diff --color -ur fastrtps-release-release-jazzy-fastrtps-2.14.4-1/src/cpp/fastdds/topic/DDSSQLFilter/DDSFilterValue.hpp fastrtps-release-release-jazzy-fastrtps-2.14.4-1-new/src/cpp/fastdds/topic/DDSSQLFilter/DDSFilterValue.hpp
--- fastrtps-release-release-jazzy-fastrtps-2.14.4-1/src/cpp/fastdds/topic/DDSSQLFilter/DDSFilterValue.hpp	2024-12-02 12:08:06.000000000 +0100
+++ fastrtps-release-release-jazzy-fastrtps-2.14.4-1-new/src/cpp/fastdds/topic/DDSSQLFilter/DDSFilterValue.hpp	2025-04-12 21:57:29.390720068 +0200
@@ -22,6 +22,8 @@
 #include <memory>
 #include <regex>
 
+#include <cstdint>
+
 #include <fastrtps/utils/fixed_size_string.hpp>
 
 namespace eprosima {
