diff -ur rosbag2-release-release-humble-rosbag2_compression_zstd-0.15.4-2/CMakeLists.txt rosbag2-release-release-humble-rosbag2_compression_zstd-0.15.4-2-new/CMakeLists.txt
--- rosbag2-release-release-humble-rosbag2_compression_zstd-0.15.4-2/CMakeLists.txt	2023-01-11 22:12:41.000000000 +0100
+++ rosbag2-release-release-humble-rosbag2_compression_zstd-0.15.4-2-new/CMakeLists.txt	2023-03-05 22:33:17.411256034 +0100
@@ -24,8 +24,8 @@
 find_package(pluginlib REQUIRED)
 find_package(rcpputils REQUIRED)
 find_package(rosbag2_compression REQUIRED)
-find_package(zstd_vendor REQUIRED)
-find_package(zstd REQUIRED)
+include(FindPkgConfig)
+pkg_check_modules(libzstd REQUIRED libzstd)
 
 add_library(${PROJECT_NAME} SHARED
   src/rosbag2_compression_zstd/compression_utils.cpp
@@ -38,7 +38,7 @@
 ament_target_dependencies(${PROJECT_NAME}
   rcpputils
   rosbag2_compression
-  zstd)
+  libzstd)
 target_compile_definitions(${PROJECT_NAME} PRIVATE ROSBAG2_COMPRESSION_ZSTD_BUILDING_DLL)
 pluginlib_export_plugin_description_file(rosbag2_compression plugin_description.xml)
 
@@ -61,7 +61,7 @@
 ament_export_targets(export_${PROJECT_NAME})
 
 # order matters here, first vendor, then zstd
-ament_export_dependencies(rcpputils rosbag2_compression zstd_vendor zstd)
+ament_export_dependencies(rcpputils rosbag2_compression libzstd)
 
 
 if(BUILD_TESTING)
diff -ur rosbag2-release-release-humble-rosbag2_compression_zstd-0.15.4-2/package.xml rosbag2-release-release-humble-rosbag2_compression_zstd-0.15.4-2-new/package.xml
--- rosbag2-release-release-humble-rosbag2_compression_zstd-0.15.4-2/package.xml	2023-01-11 22:12:41.000000000 +0100
+++ rosbag2-release-release-humble-rosbag2_compression_zstd-0.15.4-2-new/package.xml	2023-03-05 22:12:30.938810129 +0100
@@ -17,7 +17,6 @@
   <depend>rcpputils</depend>
   <depend>rcutils</depend>
   <depend>rosbag2_compression</depend>
-  <depend>zstd_vendor</depend>
 
   <test_depend>ament_cmake_gmock</test_depend>
   <test_depend>ament_lint_auto</test_depend>
